src/index.ts

```diff
@@ -125,7 +125,12 @@ const startIPCServer = async (socketPath: string, project: Project) => {
 };
 
 const childProcessArgs = () => {
-  return ["-r", path.join(__dirname, "child-process-registration.js"), "--enable-source-maps"];
+  return [
+    "-r",
+    path.join(__dirname, "child-process-registration.js"),
+    "-r",
+    path.join(__dirname, "../node_modules/@cspotcode/source-map-support/register"),
```

Comment:
A slightly node-ier way to get this path would be to do `require.resolve('@cspotcode/source-map-support')` which uses the node module resolution algorithm and would play nice with other fancy package managers like `pnpm` and whatnot in the future I think.